packages/utils/package.json

```
@@ -0,0 +1,66 @@
+{
+  "name": "@typescript-eslint/utils",
+  "version": "5.8.1",
```

this will fix the CI job

```suggestion
  "version": "5.9.1",
```